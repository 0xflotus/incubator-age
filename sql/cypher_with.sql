LOAD 'agensgraph';
SET search_path TO ag_catalog;

SELECT create_graph('cypher_with');

SELECT * FROM cypher('cypher_with', $$
WITH true AS b
RETURN b
$$) AS (b bool);

-- Expression item must be aliased.
SELECT * FROM cypher('cypher_with', $$
WITH 1 + 1
RETURN i
$$) AS (i int);

SELECT drop_graph('cypher_with');
