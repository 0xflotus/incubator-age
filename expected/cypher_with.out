LOAD 'agensgraph';
SET search_path TO ag_catalog;
SELECT create_graph('cypher_with');
NOTICE:  graph "cypher_with" has been created
 create_graph 
--------------
 
(1 row)

SELECT * FROM cypher('cypher_with', $$
WITH true AS b
RETURN b
$$) AS (b bool);
 b 
---
 t
(1 row)

-- Expression item must be aliased.
SELECT * FROM cypher('cypher_with', $$
WITH 1 + 1
RETURN i
$$) AS (i int);
ERROR:  expression item must be aliased
LINE 2: WITH 1 + 1
             ^
HINT:  Items can be aliased by using AS.
SELECT drop_graph('cypher_with');
NOTICE:  graph "cypher_with" has been dropped
 drop_graph 
------------
 
(1 row)

